template/README.md

```diff
@@ -182,20 +203,39 @@ vagrant ssh -c 'vault read consul/creds/dev-team'
 vagrant ssh -c 'vault read consul/creds/admin-team'
 ```
 
-*Tokens can be used to access UI (different access level depends on role)
+> :bulb: Tokens can be used to access UI (different access level depends on role)
+
+
+## Usage
+### Commands
+There are several commands that help to run the vagrant-box:
+- `make install` installs all prerequisites
+
+- `make up` provisions a [vagrant-hashistack](https://github.com/fredrikhgrelland/vagrant-hashistack/) box on your machine. After the machine and hashistack are set up it will run the [Default Startup Scheme](#default-startup-scheme).
```

Comment:
```suggestion
- `make up` provisions a [vagrant-hashistack](https://github.com/fredrikhgrelland/vagrant-hashistack/) box on your machine, deploy Terraform module, and run all tests locally.
```

Reply:
Don't agree with this one. It does not necessarily deploy the terraform module, that's up to the user (at least the way we have it now).

Reply:
I guess ref to #243